script.js

```diff
@@ -22,13 +23,109 @@ let timer;
 let seconds = 60;
 let userName;
 
+const requestUrl = 'https://opentdb.com/api.php?amount=5&type=multiple';
+const preloader = document.querySelector('.preloader');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/voronkova-ab/-game-Who-want-to-be-a-millionaire-/pullRequest?prid=7740148)
```suggestion
const preloader = document.querySelector(".preloader");
```